docs/resources/oceancd_verification_template.md

```diff
@@ -0,0 +1,246 @@
+          new_relic_query   = "FROM Metric SELECT count"
+        }
+
+        cloud_watch {
+          duration = "5m"
+          metric_data_queries {
+            id = "utilization"
+            metric_stat {
+              metric {
+                metric_name = "Test"
+                namespace   = "AWS/EC2"
+                dimensions {
+                  dimension_name = "instandId"
+                  dimension_value          = "i-123044"
+                }
+              }
+
+              metric_period = 400
+              stat          = "average"
+              unit          = "None"
+            }
+
+            expression  = "SELECT AVG(CPUUtilization) FROM SCHEMA"
+            label       = "TestLabel"
+            return_data = false
+            period      = 300
+          }
+        }
+
+        web {
+          method  = "GET"
+          url     = "https://oceancd.com/api/v1/metrics?clusterId= args.clusterId"
+          headers {
+            web_header_key   = "Autorization"
+            web_header_value = "Bearer=args.token"
+          }
+          body            = "{\"key\": \"test\"}"
+          timeout_seconds = 20
+          json_path       = "$.data"
+          insecure        = false
+        }
+
+        job {
+          spec {
+            backoff_limit = 1
+            job_template {
+              template_spec {
+                containers {
+                  container_name = "hello"
+                  command = ["sh","-c"]
+                  image    = "nginx.2.1"
+                }
+                restart_policy = "never"
+              }
+            }
+          }
+        }
+          jenkins {
+                pipeline_name     = "testPipelineName"
+                tls_verification  = true
+                timeout           = "2m"
+                jenkins_interval  = "5s"
+                jenkins_parameters {
+                  parameter_key   = "app"
+                  parameter_value = "my-app"
+                }
+            }
+        
+      }
+      baseline {
+        baseline_provider {
+           prometheus {
+             prometheus_query = "http_requests_total.status!"
+           }
+
+           datadog {
+             duration = "2m"
+             datadog_query    = "avg:kubernetes.cpu.user"
+           }
+
+           new_relic {
+             profile = "test"
+             new_relic_query   = "FROM Metric SELECT count*"
+           }
+        }
+        min_range = 40
+        max_range = 50
+        threshold = "range"
+      }
+  }
+ // ----------------------------------------------------------------------------
+}
+
+```
+
+```
+output "name" {
+  value = spotinst_oceancd_verification_template.example.name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) Identifier name for Ocean CD Verification Template. Must be unique.
+* `args` - (Optional) List of verification arguments. You may specify either value OR valueFrom but not both.
```

Comment:
please highlight vale and valueFrom and add the condition which is available in the document

Reply:
highlighted and added condition